### DISCUSSION TITLE
Have way to not emulate layout

### DISCUSSION THREAD

**Discussion by roubaobaozi** [nvlgzr/fngrng#3] [Your Carefully Crafted Thoughts]
Those of us with programmable boards may want to show our layout on screen, but not have the site emulate it, as it then makes it incorrect

**Comment by nvlgzr** [comment]
Totally understand. I've got a Moonlander, myself.

Have you tried toggling _Imitate Layout_ mode using Ctrl-I (^I)?

**Comment by nvlgzr** [comment]
![Screenshot showing location of Imitate Layout toggle. If you're reading this with a screen reader, please know that I'm interested in making this tool more accessible, and welcome your feedback. Cheers! —Elbong](https://user-images.githubusercontent.com/88254481/155789790-acd0e80f-beed-4bcf-8e55-e6c02007746d.png)

**Comment by bclnr** [comment]
Thanks for that, I had the same issue. For the record, until reading this discussion I missed the option because I didn't discover that they appeared when hovering the keyboard.
Maybe it would help discoverability to make all options appear when you hover any of the elements.

**Comment by nvlgzr** [comment]
This is excellent feedback. Thanks, @bclnr!

I actually _started_ with all controls displaying together, but it turned out to result in an overwhelming visual explosion every time you so much as nudge the mouse/trackpad. 😬 That said, it's clearly too easy to miss the "Imitate Layout" toggle, even after folks have discovered the whole "invisible controls" thing.

With all that in mind, I've created #10 in hopes of making this better for future users.
